android/app/src/main/res/layout/search_item_speaker.xml

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
IMO the problem with the old layout is that it fits a GridLayout Manager much better rather than it would with a LinearLayout Manager. And to have have different layout managers for the individual lists would be a problem as we would have to make multiple Recycler Views or we would have to set custom spans in the StaggeredLayoutManager.
The layout that I have made has the exact same resource ID's so nothing would be broken. Its just a restructured Layout with a different name.